.circleci/config.yml

```
@@ -0,0 +1,118 @@
+version: 2.1
+jobs:
+  initialize: # Initialize review app on pr
+    docker:
+      - image: cimg/python:3.6
+    steps:
+      - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - restore_cache: # Note: Delete cache manually by updating CACHE_VERSION in Project Settings.
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+      - run:
+          name: Install python dependencies in virtual environment
+          command: |
+            cd .circleci/
+            python3 -m venv venv
+            source venv/bin/activate
+            pip install --upgrade pip
+            pip install -r ci-requirements.txt
+      - run:
+          name: Run Initialize command
+          command: |
+            cd .circleci/
+            env
+            source venv/bin/activate
+            python3 initialize.py
+      - save_cache:
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+          paths:
+            - "venv"
+  deploy: # Deploy code to main app on merge
+    docker: 
+      - image: cimg/python:3.6
+    steps:
+      - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - restore_cache:
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+      - run:
+          name: Install python dependencies in virtual environment
+          command: |
+            cd .circleci/
+            python3 -m venv venv
+            source venv/bin/activate
+            pip install --upgrade pip
+            pip install -r ci-requirements.txt
+      - run:
+          name: Run Deploy Command
+          command: |
+            cd .circleci/
+            env
+            source venv/bin/activate
+            python3 deploy.py
+      - save_cache:
+          key: deps1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum ".circleci/ci-requirements.txt" }}
+          paths:
+            - "venv"
+  delete: # Delete old review apps on schedule or on merge
```

```suggestion
  delete: # Delete old review apps on schedule
```

AFAIK, we aren't deleting on merge, we're only deleting on a schedule. This is because we didn't find a CI hook for "on merge". And that's OK